no-OS/zynqmp_ddr_patch.sh

```diff
@@ -0,0 +1,135 @@
+	"# PSU_DDRC_ADDRMAP1_ADDRMAP_BANK_B0                                               0x9"
+	"#(OFFSET, MASK, VALUE)      (0XFD070204, 0x001F1F1FU ,0x001F0909U)  */"
+	"mask_write 0XFD070204 0x001F1F1F 0x001F0909"
+	"# PSU_DDRC_ADDRMAP5_ADDRMAP_ROW_B11                                               0x7"
+	"# PSU_DDRC_ADDRMAP5_ADDRMAP_ROW_B1                                                0x7"
+	"# PSU_DDRC_ADDRMAP5_ADDRMAP_ROW_B0                                                0x7"
+	"#(OFFSET, MASK, VALUE)      (0XFD070214, 0x0F0F0F0FU ,0x070F0707U)  */"
+	"mask_write 0XFD070214 0x0F0F0F0F 0x070F0707"
+	"# PSU_DDRC_ADDRMAP6_ADDRMAP_ROW_B15                                               0x7"
+	"# PSU_DDRC_ADDRMAP6_ADDRMAP_ROW_B14                                               0x7"
+	"# PSU_DDRC_ADDRMAP6_ADDRMAP_ROW_B13                                               0x7"
+	"# PSU_DDRC_ADDRMAP6_ADDRMAP_ROW_B12                                               0x7"
+	"#(OFFSET, MASK, VALUE)      (0XFD070218, 0x8F0F0F0FU ,0x07070707U)  */"
+	"mask_write 0XFD070218 0x8F0F0F0F 0x07070707"
+	"# PSU_DDRC_ADDRMAP8_ADDRMAP_BG_B1                                                 0x1f"
+	"#(OFFSET, MASK, VALUE)      (0XFD070220, 0x00001F1FU ,0x00001F08U)  */"
+	"mask_write 0XFD070220 0x00001F1F 0x00001F08"
+	"# PSU_DDRC_ADDRMAP9_ADDRMAP_ROW_B5                                                0x7"
+	"# PSU_DDRC_ADDRMAP9_ADDRMAP_ROW_B4                                                0x7"
+	"# PSU_DDRC_ADDRMAP9_ADDRMAP_ROW_B3                                                0x7"
+	"# PSU_DDRC_ADDRMAP9_ADDRMAP_ROW_B2                                                0x7"
+	"#(OFFSET, MASK, VALUE)      (0XFD070224, 0x0F0F0F0FU ,0x07070707U)  */"
+	"mask_write 0XFD070224 0x0F0F0F0F 0x07070707"
+	"# PSU_DDRC_ADDRMAP10_ADDRMAP_ROW_B9                                               0x7"
+	"# PSU_DDRC_ADDRMAP10_ADDRMAP_ROW_B8                                               0x7"
+	"# PSU_DDRC_ADDRMAP10_ADDRMAP_ROW_B7                                               0x7"
+	"# PSU_DDRC_ADDRMAP10_ADDRMAP_ROW_B6                                               0x7"
+	"#(OFFSET, MASK, VALUE)      (0XFD070228, 0x0F0F0F0FU ,0x07070707U)  */"
+	"mask_write 0XFD070228 0x0F0F0F0F 0x07070707"
+	"# PSU_DDRC_ADDRMAP11_ADDRMAP_ROW_B10                                              0x7"
+	"#(OFFSET, MASK, VALUE)      (0XFD07022C, 0x0000000FU ,0x00000007U)  */"
+	"mask_write 0XFD07022C 0x0000000F 0x00000007"
+	"# PSU_DDR_PHY_DTPR0_TRRD                                                          0x7"
+	"#(OFFSET, MASK, VALUE)      (0XFD080110, 0xFFFFFFFFU ,0x07240F08U)  */"
+	"mask_write 0XFD080110 0xFFFFFFFF 0x07240F08"
+	"# PSU_DDR_PHY_DTPR4_TRFC                                                          0x176"
+	"#(OFFSET, MASK, VALUE)      (0XFD080120, 0xFFFFFFFFU ,0x01762B07U)  */"
+	"mask_write 0XFD080120 0xFFFFFFFF 0x01762B07")
+
+if [[( -z "$1" ) || ($1 != *"psu_init.tcl") ]];
```

Comment:
you can also do something like `file_to_patch=${1:-psu_init.tcl}`
if no file is specified, just pass the default;
you don't need to tell the user to make it a specific file; we can let him/her decide